### PR TITLE
feat(contributing): edit contributing doc

### PR DESCRIPTION
**Pull request by gregorycat**
MP-382

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,92 @@
...
+## Submission guidelines
+
+### Submitting an issue
+
+Before you submit your issue search the archive, maybe your question was already answered.
+
+If your issue appears to be a bug, and hasn't been reported, open a new issue. Help us to maximize the effort by not reporting duplicate issues. Providing the following information will increase the chances of your issue being dealt with quickly:
+
+-   **Motivation for or Use Case** - explain why this is a bug for you
+-   **LumApps Extensions Templates Versiob** - is it a regression?
```
Comment (germainruget) [inline CONTRIBUTING.md:32]: Typo on version

**Review by germainruget — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,92 @@
...
+    - [Submitting an issue](#submitting-an-issue)
+    - [Submitting a pull request](#submitting-a-pull-request)
+  - [Coding rules](#coding-rules)
+  - [Git commit guidelines](#git-commit-guidelines)
+  - [Publish](#publish)
+
+## Found an issue?
+If you find a bug in the source code or a mistake in the documentation, you can help us by submitting an issue to our [GitHub Repository](https://github.com/lumapps/lumapps-extensions-templates/issues).
+
+If you are feeling it, you can even fix the issue yourself and submit a Pull Request. Before opening a Pull Request, please see the Submission Guidelines below.
```
Comment (theo-ardouin) [inline CONTRIBUTING.md:18]: `If you feel like it` would be better imo
`you can fix the issue and submit` (`yourself` is a bit too repetitive)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,4 +1,92 @@
...
+
+## Want a feature?
+You can request a new feature by submitting an issue to our [GitHub Repository](https://github.com/lumapps/lumapps-extensions-templates/issues). If you would like to implement a new feature then consider what kind of change it is, discuss it with us before hand in your issue, so that we can better coordinate our efforts, prevent duplication of work, and help you to craft the change so that it is successfully accepted into the project.
```
Comment (theo-ardouin) [inline CONTRIBUTING.md:21]: `before handing in ...`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,4 +1,92 @@
...
+
+## Submission guidelines
+
+### Submitting an issue
+
+Before you submit your issue search the archive, maybe your question was already answered.
+
+If your issue appears to be a bug, and hasn't been reported, open a new issue. Help us to maximize the effort by not reporting duplicate issues. Providing the following information will increase the chances of your issue being dealt with quickly:
```
Comment (theo-ardouin) [inline CONTRIBUTING.md:29]: `Help us to _minimize our_ efforts`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,4 +1,92 @@
...
+
+```bash
+git fetch && git rebase origin/master && git push --force-with-lease
+```
+
+That's it! Thank you for your contribution!
+
+## Coding rules
+
+We're using ES6 JavaScript, TypeScript and to build the tool.
```
Comment (theo-ardouin) [inline CONTRIBUTING.md:73]: `TypeScript and ??? to build ..` Missing a word?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,4 +1,92 @@
-# Contributing
+# Contributing to LumApps Extensions Templates
+
+<h2>Table of contents</h2>
```
Comment (theo-ardouin) [inline CONTRIBUTING.md:3]: ## ?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,4 +1,92 @@
...
+That's it! Thank you for your contribution!
+
+## Coding rules
+
+We're using ES6 JavaScript, TypeScript and to build the tool.
+NPM scripts are used to ease the setup, start and build of LumApps Extensions Template.
+
+The coding convention is the following:
+
+-   4 spaces for indentation, for JavaScript, TypeScript.
```
Comment (theo-ardouin) [inline CONTRIBUTING.md:78]: Extra `,` after `indentation`

File: CONTRIBUTING.md (hunk 6 above)
Comment (gregorycat) [reply CONTRIBUTING.md:3]: With ## the `Table of contents` is added to the table of content, and I don't want this

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,4 +1,92 @@
...
+    - [Submitting an issue](#submitting-an-issue)
+    - [Submitting a pull request](#submitting-a-pull-request)
+  - [Coding rules](#coding-rules)
+  - [Git commit guidelines](#git-commit-guidelines)
+  - [Publish](#publish)
+
+## Found an issue?
+If you find a bug in the source code or a mistake in the documentation, you can help us by submitting an issue to our [GitHub Repository][Github Repo].
+
+If you are feeling it, you can even fix the issue and submit a Pull Request. Before handing in a Pull Request, please see the Submission Guidelines below.
```
Comment (theo-ardouin) [inline CONTRIBUTING.md:18]: Still should be `If you feel like it`, otherwise, it makes no sense

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,4 +1,92 @@
...
+## Want a feature?
+You can request a new feature by submitting an issue to our [GitHub Repository][Github Repo]. If you would like to implement a new feature then consider what kind of change it is, discuss it with us before hand in your issue, so that we can better coordinate our efforts, prevent duplication of work, and help you to craft the change so that it is successfully accepted into the project.
+
+## Submission guidelines
+
+### Submitting an issue
+
+Before you submit your issue search the archive, maybe your question was already answered.
+
+If your issue appears to be a bug, and hasn't been reported, open a new issue. Help us to minimize our effort by not reporting duplicate issues. Providing the following information will increase the chances of your issue being dealt with quickly:
```
Comment (theo-ardouin) [inline CONTRIBUTING.md:29]: our effort`s`

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,4 +1,92 @@
...
+
+```bash
+git fetch && git rebase origin/master && git push --force-with-lease
+```
+
+That's it! Thank you for your contribution!
+
+## Coding rules
+
+We're using ES6 JavaScript and TypeScript to build the tool.
```
Comment (theo-ardouin) [inline CONTRIBUTING.md:73]: What do you mean by `build the tool`?

Comment (gregorycat) [reply CONTRIBUTING.md:73]: It means that the tools have been developed using these technologies, and contributors have to use these techno in their implementations.

**Review by theo-ardouin — APPROVED** [review]
Ok

**Pull request merged**